books.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def ad_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
+end
```

Comment:
To create a rental here we need the person and date and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association

```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```